frontends/web/src/utils/trailing-zeroes.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2022 Shift Devices AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export function removeTrailingZeroes(amount: string, unit: string): string {
+  if (amount.includes('.')) {
+    switch (unit) {
+    case 'BTC':
+    case 'TBTC':
+    case 'LTC':
+    case 'TLTC':
+      return amount.replace(/\.?0+$/,'');
```

Comment:
the same issue as before exists. `10` turns to `1`. 

check the Go implementation of how to strip zeroes, it considers all the edge cases.